contract-ui/tabs/code/components/code-overview.tsx

```diff
@@ -627,11 +627,11 @@ export const CodeOverview: React.FC<CodeOverviewProps> = ({
             </Text>
           </Flex>
         </Flex>
-        {!onlyInstall ? (
+        {true ? (
           <>
-            {foundExtensions.length > 0 ? (
+            {true ? (
```

Comment:
can we simplify this? there's unreachable code here

Reply:
Ah yeah my bad, was testing and committed accidently